llama_index/chat_engine/types.py

```diff
@@ -168,14 +200,12 @@ async def astream_chat(
 
     def chat_repl(self) -> None:
         """Enter interactive chat REPL."""
-        print("===== Entering Chat REPL =====")
-        print('Type "exit" to exit.\n')
```

Comment:
Any reason to remove this?

Reply:
Yikes - I think I used some brute force `print` when debugging something, and deleted these lines with `print` by accident. Restored.